birdhouse/config/thredds/default.env

```diff
@@ -19,3 +37,13 @@ OPTIONAL_VARS="
   \$THREDDS_DOCKER
   \$THREDDS_VERSION
 "
+
+export DELAYED_EVAL="
+  $DELAYED_EVAL
+  DATA_LOCATION_NAME
+  DATA_LOCATION_FOR_THREDDS_ON_CONTAINER
+  DATA_LOCATION_FOR_THREDDS_ON_HOST
+  RAW_DATA_LOCATION_NAME
+  RAW_DATA_LOCATION_FOR_THREDDS_ON_CONTAINER
+  RAW_DATA_LOCATION_FOR_THREDDS_ON_HOST
```

Comment:
Missing all the `_lower` vars in `DELAYED_EVAL`. 

Basically all variables that depend on other vars need to be in `DELAYED_EVAL`.

So `DATA_LOCATION_NAME` and `DATA_LOCATION_FOR_THREDDS_ON_CONTAINER`, and their `RAW_` counterparts, do not need to be in `DELAYED_EVAL` since they do not depend on any other variables.